src/cluster_legacy.c

```diff
@@ -5807,6 +5806,14 @@ int clusterNodeIsPrimary(clusterNode *n) {
     return n->flags & CLUSTER_NODE_PRIMARY;
 }
 
+/* Only primary that own slots have the voting rights.
+ * And only voting primary will be counted in cluster size.
+ *
+ * Returns 1 if the node has voting rights, otherwise returns 0. */
+int clusterNodeIsVotingPrimary(clusterNode *n) {
```

Comment:
If this function is only used inside this file, it can be `static inline` and doesn't need to be in the .h file.